nomad/serf_test.go

```diff
@@ -446,7 +433,7 @@ func TestNomad_BadExpect(t *testing.T) {
 	testutil.WaitForResult(func() (bool, error) {
 		for _, s := range servers {
 			p, _ := s.numPeers()
-			if p != 1 {
+			if p != 0 {
```

Comment:
This was a change in behavior.  When using bootstrap, the peers list remains empty until cluster is fully bootstrapped.  Previously, servers will at least recognize themselves as a peer.